models/offer.js

```diff
@@ -3,6 +3,8 @@ const {
   enums: { AUTHOR_ROLE_ENUM, SPOKEN_LANG_ENUM, SUBJECT_LEVEL_ENUM }
 } = require('~/consts/validation')
 const { USER, SUBJECT, CATEGORY, OFFER } = require('~/consts/models')
+const Category = require('./category')
+const Subject = require('./subject')
```

Comment:
Import like this require('~/models/subject')

Reply:
changed, thx